resources/views/dashboard/organization.blade.php

```diff
@@ -20,6 +20,11 @@
                 @endif
             @endcan
         @endcan
+        @if (!$user->oriented_at)
```

Comment:
This needs to actually check the organization's `oriented_at` property. You can try: 
```suggestion
        @if (!$user-> organization->oriented_at)
```